source/includes/fundamentals/code-examples/specify-query/FindAndBuilder.cs

```diff
@@ -0,0 +1,8 @@
+var builder = Builders<Guitar>.Filter;
```

Comment:
Suggestion: To help keep the examples and output in sync, you could have all examples in a single file and then use literal includes to include only the desired lines in each example. You could also generate the output blocks for code examples from the output of this one file.

Reply:
Ideally I would like to have just one file for all examples (like we do in usage examples). Unfortunately io-code-block directives don't support the :start-after: and :end-before: options yet, so I had to separate the examples out in order to use that directive.